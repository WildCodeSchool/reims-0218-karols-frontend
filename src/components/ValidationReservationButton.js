import React from "react"
import { Button, Input, FormGroup } from "reactstrap"

const ValidationReservationButton = () => {
  return (
    <FormGroup>
      <Button color="primary" size="lg" active>
        RÃ©capitulatif
      </Button>
    </FormGroup>
  )
}

export default ValidationReservationButton
