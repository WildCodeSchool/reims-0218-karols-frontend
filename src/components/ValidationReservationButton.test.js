import React from "react"
import ReactDOM from "react-dom"
import ValidationReservationButton from "./ValidationReservationButton"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ValidationReservationButton />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ValidationReservationButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
