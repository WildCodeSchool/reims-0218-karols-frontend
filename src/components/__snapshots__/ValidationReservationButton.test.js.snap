// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="form-group"
>
  <button
    className="btn btn-primary btn-lg active"
    onClick={[Function]}
    type={undefined}
  >
    RÃ©capitulatif
  </button>
</div>
`;
